dali/operators/ssd/random_crop.cc

```diff
@@ -186,6 +186,9 @@ void SSDRandomCrop<CPUBackend>::RunImpl(SampleWorkspace &ws) {
   int sample = ws.data_idx();
 
   auto N = bboxes.dim(0);
+  DALI_ENFORCE(bboxes.ndim() >= 2 && bboxes.dim(1) == 4,
```

Comment:
```suggestion
  DALI_ENFORCE(bboxes.ndim() == 2 && bboxes.dim(1) == 4,
```
???
Do we have a valid case when it's more than 2D?

Reply:
You're right